site/cypress/e2e/Cypress-TAGrading/notebook_section.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Notebook Section Testing', () => {
+    // Ignore uncaught js exceptions
+    Cypress.on('uncaught:exception', () => {
```

Comment:
Would it be better to fix the JS exceptions rather than catching and ignoring them?

Reply:
Ok i will do it